packages/utilities/src/utilities/smoothScroll/smoothScroll.js

```diff
@@ -31,9 +32,13 @@ const smoothScroll = (e, selector) => {
   } else {
     return null;
   }
-  document.querySelector(getSelector).scrollIntoView({
+
+  window.scroll({
```

Comment:
@RaphaelAmadeu are we switching from `scrollIntoView`?

Reply:
@jeffchew ,
`scrollIntoView` does not support offset, and it was causing the heading to not be shown completely on mobile devices.